OpenEdXMobile/src/main/java/org/edx/mobile/model/video/VideoQuality.kt

```diff
@@ -0,0 +1,10 @@
+package org.edx.mobile.model.video
+
+import org.edx.mobile.R
+
+enum class VideoQuality(val titleResId: Int) {
+    AUTO(R.string.auto_recommended_text),
+    OPTION_360P(R.string.video_quality_p360),
+    OPTION_540P(R.string.video_quality_p540),
+    OPTION_720P(R.string.video_quality_p720)
+}
```

Comment:
Can we add `val value: String = this.name.replace("OPTION_", "").toLowerCase()` to get option value.

Reply:
done.